xml/telnet_banners.xml

```diff
@@ -816,4 +816,680 @@
+    <!-- IRIX (artemis.biol.uoa.gr)\n\n\n\nlogin: -->
+    <example _encoding="base64" host.name="artemis.biol.uoa.gr">
+      SVJJWCAoYXJ0ZW1pcy5iaW9sLnVvYS5ncikKCgoKbG9naW46IA==
+    </example>
+    <param pos="0" name="hw.vendor" value="SGI"/>
+    <param pos="0" name="hw.family" value="IRIX"/>
+    <param pos="0" name="hw.product" value="IRIX"/>
+    <param pos="1" name="host.name"/>
+  </fingerprint>
+  <fingerprint pattern="^(?m)(RS [^\s]+) System Software, Version ([^\s]+).*">
+    <description>RiverStone router</description>
+    <!-- ++++++++++++++++++++++++++++++++++++++++\nRS 8000 System Software, Version 9.3.0.5\n
+    Riverstone Networks, Inc., Copyright (c) 2000-2004. All rights reserved.\nSystem started on 2018-10-11 22:02:03
+    \n++++++++++++++++++++++++++++++++++++++++ -->
+    <example _encoding="base64" hw.product="RS 8000" os.version="9.3.0.5">
+      LS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tL
+      S0tLS0tLQpSUyA4MDAwIFN5c3RlbSBTb2Z0d2FyZSwgVmVyc2lvbiA5LjMuMC41ClJpdmVyc3RvbmUgTmV0d2
+      9ya3MsIEluYy4sIENvcHlyaWdodCAoYykgMjAwMC0yMDA0LiBBbGwgcmlnaHRzIHJlc2VydmVkLgpTeXN0ZW0
+      gc3RhcnRlZCBvbiAyMDE4LTEwLTExIDIyOjAyOjAzCi0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0t
+      LS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS4uLg==
+    </example>
+    <param pos="0" name="hw.vendor" value="Riverstone"/>
+    <param pos="0" name="hw.family" value="Riverstone"/>
+    <param pos="0" name="hw.device" value="Router"/>
+    <param pos="1" name="hw.product"/>
+    <param pos="2" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="(ES [^\s]+) System Software, Version ([^\s]+)">
+    <description>a Riverstone router`</description>
+    <!-- +++++++++++++++++++++++++++++++++++++++\nES 10170 System Software, Version 9.3.0.4\n
+    Riverstone Networks, Inc., Copyright (c) 2000-2003. All rights reserved.\nSystem started on 2018-09-06 15:58:\n
+    +++++++++++++++++++++++++++++++++++++++ -->
+    <example _encoding="base64" hw.product="ES 10170" os.version="9.3.0.4">
+      LS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tL
+      S0tLS0tLQpFUyAxMDE3MCBTeXN0ZW0gU29mdHdhcmUsIFZlcnNpb24gOS4zLjAuNApSaXZlcnN0b25lIE5ldH
+      dvcmtzLCBJbmMuLCBDb3B5cmlnaHQgKGMpIDIwMDAtMjAwMy4gQWxsIHJpZ2h0cyByZXNlcnZlZC4KU3lzdGV
+      tIHN0YXJ0ZWQgb24gMjAxOC0wOS0wNiAxNTo1ODozMAotLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0t
+      LS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS4uLg==
+    </example>
+    <param pos="0" name="hw.vendor" value="Riverstone"/>
+    <param pos="0" name="hw.family" value="ES"/>
+    <param pos="0" name="hw.device" value="Router"/>
+    <param pos="1" name="hw.product"/>
+    <param pos="2" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="HP.*ProCurve Switch">
+    <description>HP ProCurve Switch</description>
+    <!-- ==============================================================================\nHP J4121A ProCurve Switch 4000M\n
```

Comment:
These appear to have product names in them.  Those should be extracted and tested.